pulp_rpm/app/models/repository.py

```diff
@@ -148,6 +151,61 @@ class RpmRemote(Remote):
     """
 
     TYPE = 'rpm'
+    sles_auth_token = models.CharField(
```

Comment:
@bmbouter , @dralley , should auth token be mutually exclusive with username/password or client cert/key?

Reply:
These things aren't intended to be configured together, but allowing them to be is probably ok. For example multi-factor authentication promotes the idea that two types of authentication are used at once, so if any server side decides to do that, our helpful feature would prevent users from using it.

I don't have any strong feeling on this, ^ is just my general take in this situation.